Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -595,15 +595,16 @@ void Renderer::DrawDebugText()
     ImGui::SetNextWindowPos(ImVec2(ImGui::GetIO().DisplaySize.x - (10.0f * m_backbuffer_scale),
                                    10.0f * m_backbuffer_scale),
                             ImGuiCond_Always, ImVec2(1.0f, 0.0f));
-    ImGui::SetNextWindowSize(ImVec2(100.0f * m_backbuffer_scale, 30.0f * m_backbuffer_scale));
+    ImGui::SetNextWindowSize(ImVec2(75.0f * m_backbuffer_scale, 30.0f * m_backbuffer_scale));
 
     if (ImGui::Begin("FPS", nullptr,
                      ImGuiWindowFlags_NoTitleBar | ImGuiWindowFlags_NoInputs |
                          ImGuiWindowFlags_NoMove | ImGuiWindowFlags_NoSavedSettings |
                          ImGuiWindowFlags_NoScrollbar | ImGuiWindowFlags_NoNav |
                          ImGuiWindowFlags_AlwaysAutoResize | ImGuiWindowFlags_NoFocusOnAppearing))
     {
-      ImGui::TextColored(ImVec4(0.0f, 1.0f, 1.0f, 1.0f), "FPS: %.2f", m_fps_counter.GetFPS());
+      const double fps = m_fps_counter.GetFPS();
+      ImGui::TextColored(ImVec4(0.0, 1.0, 1.0, 1.0f), "FPS: %.0lf", std::round(fps));
```

Comment:
I did initially want to use %d because it would be more optimal, but different architectures cause `long` to be different sizes which makes printf a little awkward.

Reply:
I think you can write `%ld` which corresponds to `long int` which is the same as `int`. It's a bit annoying that ImGui uses printf-style formatting, though; Dolphin mostly uses [fmt](https://fmt.dev/) which doesn't require you to exactly match the types (and can give compile-time errors if you do have a mismatched type).